rbac/management/group/view.py

```diff
@@ -59,12 +59,19 @@ def username_filter(queryset, field, value):
         filtered_set = queryset.filter(**filters)
         return filtered_set | Group.platform_default_set()
 
+    def uuid_filter(queryset, field, values):
+        """Filter for group uuid lookup."""
+        filters = {f'{field}__in': values.split(',')}
```

Comment:
We may want a UUID validation here? If I give an invalid UUID as one of the values in the query param, I get a 500.

Reply:
👍

Reply:
Added in a simple check that'll throw back a 400 and an 'invalid uuid' message.